src/switch_core_media.c

```diff
@@ -4843,58 +4843,68 @@ static switch_status_t check_ice(switch_media_handle_t *smh, switch_media_type_t
 		}
 
 
-		if (engine->rtp_session && ((val = switch_channel_get_variable(smh->session->channel,
-																	   type == SWITCH_MEDIA_TYPE_VIDEO ?
-																	   "rtcp_video_interval_msec" : "rtcp_audio_interval_msec"))
-									|| (val = type == SWITCH_MEDIA_TYPE_VIDEO ?
-										smh->mparams->rtcp_video_interval_msec : smh->mparams->rtcp_audio_interval_msec))) {
+		if (switch_channel_var_true(smh->session->channel, "telnyx_disable_rtcp") ||
```

Comment:
Is not this logic broken now? Since reverted to disable_rtcp it should check for only if it is defined and skip this part, or continue processing if it is undefined or defined to false.

Reply:
fixed to use switch_channel_var_false